eventbus/local/eventbus_test.go

```diff
@@ -39,4 +46,56 @@ func TestEventBus(t *testing.T) {
 
 	eventbus.AcceptanceTest(t, bus1, bus2, time.Second)
 
+	bus1.Close()
+	bus2.Close()
+	bus1.Wait()
+	bus2.Wait()
+}
+
+func TestCloseAndWait(t *testing.T) {
+	bus := NewEventBus(nil)
+
+	id, _ := eh.ParseUUID("c1138e5f-f6fb-4dd0-8e79-255c6c8d3756")
+	event := eh.NewEventForAggregate(mocks.EventType, &mocks.EventData{Content: "event1"}, time.Now(), mocks.AggregateType, id, 1)
+
+	repo := &mocks.Repo{}
+	slowProjector := NewSlowProjector(repo, time.Second)
+	projectorEventHandler := projector.NewEventHandler(slowProjector, repo)
+	bus.AddHandler(eh.MatchAny(), projectorEventHandler)
+
+	bus.PublishEvent(context.Background(), event)
+
+	// Event can't be done processing yet
+	assert.False(t, repo.SaveCalled)
```

Comment:
Look into the other tests for how to write this.

Reply:
fixed using stdlib